src/hooks/useOperatorOrder.ts

```diff
@@ -30,7 +30,7 @@ async function _getOrder(
   networkIdSearchListRemaining: Network[] = NETWORK_ID_SEARCH_LIST,
 ): Promise<GetOrderResult> {
   // Get order
-  const order = await getOrder({ networkId, orderId })
+  const order = await getOrder({ networkId, orderId: orderId.toLowerCase() })
```

Comment:
why not doing this in the impl? This way we dont need to be making the fix in every use of the API or hook

![image](https://user-images.githubusercontent.com/2352112/112304176-783dc400-8c9d-11eb-8774-e22808d08c7d.png)

Reply:
I wanted to keep the API "pure". Who knows, maybe we do need a case sensitive search :shrug: 

But no strong reason for that. Will update.

Reply:
Considering the cases Michel added below, I think we might want to do this somewhere else.
While only lowercase should be fine in the API, the extra cleaning/regex matching don't belong there.

I'm thinking about adding it as a hook encapsulating `useParams`, and updating the url after cleaning the id

Reply:
Done in a hook now